### PR TITLE
[MAR-3392] Training

### PR DESCRIPTION
**Pull request by malmas18**

### CODE REVIEW

File: apps/marketplace/components/BuyerTraining/BuyerTrainingResponseFormatsStage.js [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+      }}
+    />
+    <AUheadings level="2" size="md">
+      Select what sellers need to provide through the Marketplace:
+    </AUheadings>
+    <div className={styles.formats}>
+      <CheckboxDetailsField
+        model={`${props.model}.evaluationType[]`}
+        id={`response_format_proposal`}
+        name={`response_format_proposal`}
```
Comment (tobiasbrown) [inline apps/marketplace/components/BuyerTraining/BuyerTrainingResponseFormatsStage.js]: Do these need to be wrapped in `{}`?  Applies elsewhere in this file too.

File: apps/marketplace/components/BuyerTraining/BuyerTrainingSelectStage.js [hunk 2]
```diff
@@ -0,0 +1,146 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+import { actions, Form } from 'react-redux-form'
+import formProps from 'shared/form/formPropsSelector'
+import AUheading from '@gov.au/headings/lib/js/react.js'
+import SellerSelect from 'marketplace/components/SellerSelect/SellerSelect'
```
Comment (tobiasbrown) [inline apps/marketplace/components/BuyerTraining/BuyerTrainingSelectStage.js]: What about using the `ItemSelect` component?

File: apps/marketplace/components/BuyerTraining/BuyerTrainingResponseFormatsStage.js (hunk 1 above)
Comment (malmas18) [reply apps/marketplace/components/BuyerTraining/BuyerTrainingResponseFormatsStage.js]: updated

File: apps/marketplace/components/BuyerRFX/BuyerRFXResponseFormatsStage.js [hunk 3]
```diff
@@ -126,6 +126,17 @@ const BuyerRFXResponseFormatsStage = props => (
       Select any additional assessment methods:
     </AUheadings>
     <div className={styles.formats}>
+      <CheckboxDetailsField
+        model={`${props.model}.evaluationType[]`}
+        id={`response_format_interview`}
+        name={`response_format_interview`}
```
[the remark this answers is not in the document]
Comment (malmas18) [reply apps/marketplace/components/BuyerRFX/BuyerRFXResponseFormatsStage.js]: updated

File: apps/marketplace/components/BuyerTraining/BuyerTrainingIntroductionStage.js [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+      <li>
+        The{' '}
+        <a href="/api/2/r/learning-design-standards" rel="noopener noreferrer" target="_blank">
+          Learning Design Standards
+        </a>{' '}
+        can be used to help structure your training approach.
+      </li>
+      <li>
+        View the{' '}
+        <a href="/api/2/r/support-article" target="_blank">
```
[the remark this answers is not in the document]
Comment (malmas18) [reply apps/marketplace/components/BuyerTraining/BuyerTrainingIntroductionStage.js]: updated

File: apps/marketplace/components/BuyerTraining/BuyerTrainingAdditionalInformationStage.js [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+          <a href="/api/2/r/comprehensive-terms-current.pdf" rel="noopener noreferrer" target="_blank">
+            comprehensive terms
+          </a>{' '}
+          only be applied to procurements that are complex or high value. The terms will apply to your work order, in
+          addition to the Master Agreement.
+        </p>
+        <p>
+          <CheckboxDetailsField
+            model={`${this.props.model}.comprehensiveTerms`}
+            id={`comprehensiveTerms`}
```
Comment (tobiasbrown) [inline apps/marketplace/components/BuyerTraining/BuyerTrainingAdditionalInformationStage.js]:
```suggestion
            id="comprehensiveTerms"
```

File: apps/marketplace/components/BuyerTraining/BuyerTrainingAdditionalInformationStage.js [hunk 6]
```diff
@@ -0,0 +1,145 @@
...
+            name={`comprehensiveTerms`}
```
Comment (tobiasbrown) [inline apps/marketplace/components/BuyerTraining/BuyerTrainingAdditionalInformationStage.js]:
```suggestion
            name="comprehensiveTerms"
```

File: apps/marketplace/components/BuyerTraining/BuyerTrainingResponseFormatsStage.js [hunk 7]
```diff
@@ -0,0 +1,177 @@
...
+            label="Breakdown of costs"
+            value="Breakdown of costs"
+            detailsModel={props.model}
+            disabled={!props[props.model].evaluationType.includes('Written proposal')}
+            validators={{}}
+            messages={{}}
+          />
+          <CheckboxDetailsField
+            model={`${props.model}.proposalType[]`}
+            id={`proposal_casestudy`}
```
Comment (tobiasbrown) [inline apps/marketplace/components/BuyerTraining/BuyerTrainingResponseFormatsStage.js]:
```suggestion
            id="proposal_casestudy"
```

File: apps/marketplace/components/BuyerTraining/BuyerTrainingResponseFormatsStage.js [hunk 8]
```diff
@@ -0,0 +1,177 @@
...
+            name={`proposal_casestudy`}
```
Comment (tobiasbrown) [inline apps/marketplace/components/BuyerTraining/BuyerTrainingResponseFormatsStage.js]:
```suggestion
            name="proposal_casestudy"
```

File: apps/marketplace/pages/BuyerTrainingFlowPage.js [hunk 9]
```diff
@@ -0,0 +1,168 @@
...
+          }
+          return true
+        })
+
+        if (response.data.brief.status && response.data.brief.status !== 'draft') {
+          this.props.setError('You cannot edit this opportunity as you have already published it.')
+        }
+
+        if (response.data.brief.lotSlug !== 'training2') {
+          this.props.setError('You can only edit Training briefs using this flow.')
```
Comment (tobiasbrown) [inline apps/marketplace/pages/BuyerTrainingFlowPage.js]:
```suggestion
          this.props.setError('You can only edit Training opportunities using this flow.')
```

**Pull request merged**